source/includes/options-mongosqld.yaml

```diff
@@ -139,47 +173,56 @@ description: |
 default: 1000  
 ---
 program: mongosqld
-name: sampleMode
+name: schemaMode
 directive: option
-args: <[read|write]>
+args: <[custom|auto]>
 description: |
-  .. versionadded:: 2.3
+  .. versionchanged:: 2.11
 
-  Specifies whether :binary:`~bin.mongosqld` writes schema information
-  to the MongoDB database named in the :option:`--sampleSource
-  <mongosqld --sampleSource>` option or only reads previously sampled
-  schema information from it.
+     Renamed ``--sampleMode`` to ``--schemaMode``
 
-  If you use the :option:`--sampleSource <mongosqld --sampleSource>`
-  option to specify a database to store schema information, set
-  :option:`--sampleMode <mongosqld --sampleMode>` to ``write`` to
-  allow :binary:`~bin.mongosqld` to persist data in the specified
-  database.
-  
-  .. include:: /includes/fact-samplemode-write-permission.rst
+  Configures the :ref:`sampling mode <sampling-mode-chart>` of
+  :binary:`~bin.mongosqld` to either:
 
-  .. include:: /includes/sampling-ref-chart-link.rst
+  - Sample data from the MongoDB database to generate the
+    relational schema.
 
-default: read
+  - Sample and write schema information to the MongoDB database
+    named in the :option:`--schemaSource <mongosqld --schemaSource>`
+    option.
+
+  - Read previously sampled schema information to the MongoDB
+    database named in the
+    :option:`--schemaSource <mongosqld --schemaSource>` option.
+
```

Comment:
I see that the options are "custom" and "auto", but it's not clear which option does what. I'd be explicit about the behavior of both options here.

Reply:
Yeah, unless I'm missing something from the design doc, renaming these values only makes this whole thing more confusing.

Reply:
Fair point though, took a stab at clarifying this.